app/src/main/java/com/endava/myendava/fragments/FaqFragment.java

```diff
@@ -62,7 +77,11 @@ public View provideFragmentView(LayoutInflater inflater, ViewGroup parent, Bundl
     public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
         mUnbinder = ButterKnife.bind(this, view);
 
```

Comment:
don't forget about the status bar color (call method from base fragment with secondary color)